docs/rst/tutorials/core/ros2_tutorials.rst

```diff
@@ -3,6 +3,8 @@
 Vulcanexus Core Tutorials
 ==========================
 
+ROS 2 :doc:`tutorials <../../../ros2_documentation/source/Tutorials>` are provided in the ROS2 Documentation. *Vulcanexus* expands those tutorials with new ones that show user features, tools provided with *Vulcanexus* and advanced tutorials that are not explained in the ROS 2 Documentation.
```

Comment:
Please remind to break line after every sentence. 
```suggestion
A set of :doc:`ROS 2 tutorials <../../../ros2_documentation/source/Tutorials>` are provided in the ROS 2 Documentation.
*Vulcanexus* documentation adds further ROS 2 tutorials, along with middleware specific user features only available in Fast DDS.
```

Reply:
Slightly modified.